repair/repair.cc

```diff
@@ -2253,6 +2253,45 @@ future<> repair::tablet_repair_task_impl::run() {
         auto start_time = std::chrono::steady_clock::now();
         auto parent_data = get_repair_uniq_id().task_info;
         std::atomic<int> idx{1};
+
+        // Start the off strategy updater
+        std::unordered_set<gms::inet_address> participants;
+        std::unordered_set<table_id> table_ids;
+        for (auto& meta : _metas) {
+            thread::maybe_yield();
+            participants.insert(meta.neighbors.all.begin(), meta.neighbors.all.end());
+            table_ids.insert(meta.tid);
+        }
+        abort_source as;
+        auto off_strategy_updater = seastar::async([&rs, uuid = id.uuid().uuid(), &table_ids, &participants, &as] {
```

Comment:
why a thread rather than a coroutine?

Reply:
When this code was written, there was no coroutine.  This could be a coroutine but I want to keep the code for vnode and tablet the same. It is easier to refactor later.

Reply:
ah, I see it was copy-pasted from `user_requested_repair_task_impl`.